4. User Guides/Editing Messages/README.md

```diff
@@ -34,7 +34,7 @@ You can also delete a message by emptying the contents while editing and saving
 
 ## Sending emojis
 
-You can insert emojis in your messages as you want by clicking on the emoji smiley face icon on the left of the compose message box to browse or search for emojis.
+You can insert emojis in your messages by clicking on the emoji smiley face icon on the left of the compose message box to browse or search for emojis.
 
 You can also insert emojis by inputing a colon `:` in the message compose box followed by typing the emoji name.
```

Comment:
Is it not clear enough from "after you have input two characters of the emoji's name a list of matched emojis will be displayed."?